GPU/GPUTracking/dEdx/GPUdEdx.h

```diff
@@ -145,7 +145,8 @@ GPUdnii() void GPUdEdx::fillCluster(float qtot, float qmax, int padRow, unsigned
   const int region = param.tpcGeometry.GetRegion(padRow);
   z = CAMath::Abs(z);
   const float threshold = calibContainer->getZeroSupressionThreshold(slice, padRow, padPos); // TODO: Use the mean zero supresion threshold of all pads in the cluster?
-  const float qTotIn = CAMath::Clamp(qtot, calibContainer->getMinqTot(), calibContainer->getMaxqTot());
+  const float fullGainMapGain = calibContainer->getGain(slice, padRow, padPos);
+  const float qTotIn = fullGainMapGain * CAMath::Clamp(qtot, calibContainer->getMinqTot(), calibContainer->getMaxqTot());
   const float qMaxTopologyCorr = calibContainer->getTopologyCorrection(region, o2::tpc::ChargeType::Max, tanTheta, snp, z, absRelPad, relTime, threshold, qTotIn);
```

Comment:
Perhaps it would be good to add an option to the workflow and `calibContainer` to be able to use the full gain map to correct `qmax` and `qtot` and then, of course, not use it for the `qTotIn`. This would e.g. be needed in case the full gain map was not applied in the cluster finder. As e.g. for the pilot beam data.